daemon/networkdriver/bridge/driver.go

```diff
@@ -379,6 +381,13 @@ func Allocate(job *engine.Job) engine.Status {
 		return job.Error(err)
 	}
 
+	// If no explicit gateway was given, use default.
+	if requestedGateway != nil {
```

Comment:
I personally don't like unnecessary vars and elses :)
so I prefer such code looks like:
```
gateway := bridgeNetwork.IP
if requestedGateway != nil {
    gateway = requestedGateway
}
```